api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -1460,6 +1474,11 @@ type HostedClusterStatus struct {
 	// +optional
 	KubeConfig *corev1.LocalObjectReference `json:"kubeconfig,omitempty"`
 
+	// customkubeconfig is a local secret reference to the external custom kubeconfig.
+	// Once the hypershift operator sets this status field, it will generate a secret with the specified name containing a kubeconfig within the `HostedCluster` namespace.
+	// +optional
+	CustomKubeConfig *corev1.LocalObjectReference `json:"customkubeconfig,omitempty"`
```

Comment:
the marker has a typo, should be customKubeconfig

Reply:
Humm following the above pattern of `KubeConfig` I've set CustomKubeconfig. Is that alright?